x-pack/plugins/security_solution/public/common/components/top_n/index.tsx

```diff
@@ -8,11 +8,11 @@
 import React, { useMemo } from 'react';
 import { connect, ConnectedProps } from 'react-redux';
 
-import { DataViewBase, Filter, Query } from '@kbn/es-query';
+import type { DataViewBase, Filter, Query } from '@kbn/es-query';
 import { useGlobalTime } from '../../containers/use_global_time';
 import { BrowserFields } from '../../containers/source';
 import { useKibana } from '../../lib/kibana';
-import { esQuery } from '../../../../../../../src/plugins/data/public';
+import { getEsQueryConfig } from '../../../../../../../src/plugins/data/common/es_query/get_es_query_config';
```

Comment:
Nice, this to avoid pulling in more than necessary here?

Reply:
haha, yep! Everywhere I can I try to use it from `common` to avoid more of the glut of imports which are consuming memory and hampering performance with regards to jest tests.

Reply:
Note I had to change this slightly to be: `../../../../../../../src/plugins/data/common` for the build to work but still I try to pull from `common` first before I pull from `plugins/data/public` or `plugins/data/server` to reduce the performance hit and memory leaks that exist right now from Kibana core when we pull from those locations from jest tests.